### ISSUE TITLE
After build -algorithm Fatal error

### ISSUE THREAD

**Issue by DadSimen** [biometrics/openbr#468]
Hello,
when i was trying tutorial first line for opening webcam works perfectly. Second example give me error:
```

C:\Users\JIO>br -gui -algorithm "Cvt(Gray)+Show(false)" -enroll 0.webcam
Set algorithm to Cvt(Gray)+Show(false)
Fatal: Failed to set br::Transform* br::IndependentTransform::transform to:
  SDK Path: C:/openbr
  File:
  Function:
  Line: 0
```

Any sugestions ?

**Comment by d1xlord** [comment]
Hi, having this same issue. Any solutions regarding this?

**Comment by DadSimen** [comment]
Hello,

I am trying to make it working but no success here. I find out that problem is probably in Qt. I am using wrong version of Qt 5.6 because of Windows 10 and i should use Qt 5.4 but there is no support for win10. Right now i am trying to use Qt 5.5 – i found somewhere build of Qt5.5 for win10 but still no success.

**Comment by cristobaldobranco** [comment]
I'm seeing the same issue with Qt 5.6 for Android in a compilation of OpenBR for Android. For me it was possible to switch to Qt 5.5.1 for Android and get over this stage.

**Comment by DepthDeluxe** [comment]
I've had issues with Qt 5.6 on Arch Linux x64.  I think there is some sort of change in API that is causing the problem.  Switching to Qt 5.5.1 solved the problem

**Comment by sklum** [comment]
I'm going to try to solve this issue when I get some free time.  However, if anyone has made local modifications to openbr to resolve compatibility with Qt 5.6 we would welcome them in a PR.

**Comment by d1xlord** [comment]
I also got over this issue by switching to Qt 5.4

Thanks for the reply

**Comment by raomshauty** [comment]
Hi -- I ran into this with compile openbr against qt 5.7.  I spent time looking for solution and came up with this change -- not super sure if its the right fix because I cannot understand exactly why its a problem with new qt version but not with old one ... but you can try and see if its working for you.

In method Transform::make within openbr_plugin.cpp -- change usage of qVariantFromValue<void*>(transform) to QVariant::fromValue(transform).

old
```
// causes QObject::setProperty to return false inside Object::setProperty  
        independent.set("transform", qVariantFromValue<void*>(transform));
```

new
```
// causes QObject::setProperty to return true as expected -- seems to work
        independent.set("transform", QVariant::fromValue(transform));
```

If anyone can explain exactly what goes on here, would like to read it.  I guess its something with how QObject treats c++ type information differently when using c++11 vs earlier compiler versions?

**Comment by raomshauty** [comment]
I thought to follow up - I've been using the above patch for a few days and haven't noticed any issues with it and qt 5.7 yet.

**Comment by jklontz** [comment]
Fixed by #486

**Comment by CodingAnarchy** [comment]
I ran into this issue with Qt 5.10.1, so I'm having to downgrade.  Just thought I would throw it out there in case anyone has time to fix.
